package.json

```diff
@@ -91,6 +91,7 @@
 		"@types/xml2js": "^0.4.11",
 		"jest": "^29.4.3",
 		"open-cli": "^7.1.0",
+		"pkg-pr-new": "^0.0.50",
```

Comment:
I dont think this needs to be here. The workflow is using `yarn dlx` which will download a temporary version of the package, and I think it will ignore this version entirely. So this being here serves no purpose?